allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -66,67 +66,167 @@ def __init__(
         max_length: Optional[int] = None,
         stride: int = 0,
         truncation_strategy: str = "longest_first",
-        calculate_character_offsets: bool = False,
         tokenizer_kwargs: Optional[Dict[str, Any]] = None,
     ) -> None:
         if tokenizer_kwargs is None:
             tokenizer_kwargs = {}
         else:
             tokenizer_kwargs = tokenizer_kwargs.copy()
-        if "use_fast" in tokenizer_kwargs:
-            if tokenizer_kwargs["use_fast"]:
-                logger.warning(
-                    "Fast huggingface tokenizers are known to break in certain scenarios."
-                )
-        else:
-            tokenizer_kwargs["use_fast"] = False
-        # As of transformers==2.8.0, fast tokenizers are broken.
-        self.tokenizer = AutoTokenizer.from_pretrained(model_name, **tokenizer_kwargs)
+        if "use_fast" not in tokenizer_kwargs:
+            tokenizer_kwargs["use_fast"] = True
```

Comment:
This is a nice readability trick:

```suggestion
        tokenizer_kwargs.setdefault("use_fast", True)
```

Reply:
Not sure it's more readable, but I like concise :-)